### DISCUSSION TITLE
最近风控是不是变严了，不常用的qq都会提示网络不稳定

### DISCUSSION THREAD

**Discussion by bingliran** [NapNeko/NapCatQQ#653] [General]
![image](https://github.com/user-attachments/assets/d66cdd59-988d-4480-909e-3753b5db3b8c)

我用vpn挂到服务器上ip一致也会提示这个，但是经常使用的大号挂不挂vpn都不会出现这个问题
之前这个问题也有，通常挂个vpn就没问题了，现在还有什么好的解决办法没

**Comment by Shua-github** [comment]
是的，腾讯冲业绩

**Comment by pk5ls20** [comment]
#659
